pkg/noun/noun.h

```diff
@@ -15,6 +15,7 @@
 #include "types.h"
 #include "vortex.h"
 #include "zave.h"
+#include "events.h"
```

Comment:
`events.h` is intentionally excluded here, it's an internal module and should only be imported/called explicitly where needed.

Reply:
Ok. I was using it to include the `u3e_backup` function. I've moved the `#include` directive to `disk.c` instead-- better?